src/coreclr/tools/Common/CommandLine/ArgumentSyntax.cs

```diff
@@ -107,12 +108,12 @@ public void ReportError(string message)
         public ArgumentCommand<T> DefineCommand<T>(string name, T value)
         {
             if (string.IsNullOrEmpty(name))
-                throw new ArgumentException(Strings.NameMissing, "name");
+                throw new ArgumentException(Strings.NameMissing, nameString);
```

Comment:
Is there a problem with using `nameof(name)` ?

Reply:
No, but since it's used more than once I preferred the constant fix

Reply:
The canonical pattern for argument names like this is to use `nameof` everywhere and not worry about the duplication.

You are not saving anything by using a constant string. The compiler will store the constant string just once in the binary, even when it appears multiple times in the source code.

Reply:
I'm also not sure what will it display in the scenarios where there is a parameter "name" as input which is empty. I'm afraid in those cases it will create an Argument exception with an empty string instead of the string "name"

Reply:
Nevermind it should work fine, fixing it